pay-api/src/pay_api/utils/statement_dtos.py

```diff
@@ -0,0 +1,432 @@
+            InvoiceStatus.REFUND_REQUESTED.value,
+            InvoiceStatus.REFUNDED.value,
+            InvoiceStatus.COMPLETED.value,
+        }
+
+        if status_code in default_statuses:
+            return True
+
+        match payment_method:
+            case PaymentMethod.PAD.value:
+                return status_code in {
+                    InvoiceStatus.APPROVED.value,
+                    InvoiceStatus.SETTLEMENT_SCHEDULED.value,
+                }
+
+            case PaymentMethod.EFT.value:
+                return status_code in {
+                    InvoiceStatus.APPROVED.value,
+                    InvoiceStatus.OVERDUE.value,
+                }
+
+            case PaymentMethod.EJV.value:
+                return status_code in {
+                    InvoiceStatus.APPROVED.value,
+                }
+
+            case PaymentMethod.INTERNAL.value:
+                return status_code in {
+                    InvoiceStatus.APPROVED.value,
+                }
+
+            case _:
+                return False
+
+    @classmethod
+    def from_orm(
+        cls,
+        invoice: InvoiceSearchModel,
+        payment_method: str,
+        statement_to_date: datetime,
+    ) -> StatementTransactionDTO:
+        """Create DTO from ORM invoice object for PDF statement."""
+        products = [item.description for item in invoice.payment_line_items]
+
+        details = [f"{d.get('label')} {d.get('value')}" for d in (invoice.details or [])]
+
+        # statutory fee
+        fee = invoice.total - invoice.service_fees - (invoice.gst or 0)
+
+        status_code = invoice.invoice_status_code
+
+        service_provided = cls.determine_service_provision_status(status_code, payment_method)
+
+        line_items = [PaymentLineItemSearchModel.from_row(item) for item in invoice.payment_line_items]
+
+        applied_credits = None
+        if invoice.applied_credits:
+            filtered_credits = [c for c in invoice.applied_credits if c.created_on <= statement_to_date]
+            if filtered_credits:
+                applied_credits = [AppliedCreditsSearchModel.from_row(c) for c in filtered_credits]
+
+        return cls(
+            id=invoice.id,
+            products=products,
+            details=details,
+            folio=invoice.folio_number or "-",
+            created_on=Converter().structure(invoice.created_on, FullMonthDateStr),
+            fee=Converter().structure(fee, CurrencyStr),
+            service_fee=Converter().structure(invoice.service_fees, CurrencyStr),
+            gst=Converter().structure(invoice.gst, CurrencyStr),
+            total=Converter().structure(invoice.total, CurrencyStr),
+            status_code=status_code,
+            service_provided=service_provided,
+            line_items=line_items,
+            payment_date=Converter().structure(invoice.payment_date, FullMonthDateStr),
+            refund_date=Converter().structure(invoice.refund_date, FullMonthDateStr),
```

Comment:
You don't need to do this

just do:

`created_on=invoice.created_on `

when you serialize the entire object it will become the format you need